core-customize/tccore/src/com/taloscommerce/core/event/SendOldProductInWishListEventListener.java

```diff
@@ -0,0 +1,46 @@
+package com.taloscommerce.core.event;
+
+import com.taloscommerce.core.model.SendOldProductInWishListProcessModel;
+import de.hybris.platform.core.model.product.ProductModel;
+import de.hybris.platform.processengine.BusinessProcessService;
+import de.hybris.platform.servicelayer.event.impl.AbstractEventListener;
+import de.hybris.platform.servicelayer.model.ModelService;
+
+
+import javax.annotation.Resource;
+import java.util.List;
+
+public class SendOldProductInWishListEventListener extends AbstractEventListener<SendOldProductInWishListEvent> {
+
+
+    private ModelService modelService;
+
+    private BusinessProcessService businessProcessService;
+
+
+    @Override
+    protected void onEvent(final SendOldProductInWishListEvent sendOldProductInWishListEvent) {
+        final List<ProductModel> products = sendOldProductInWishListEvent.getProducts();
```

Comment:
I recommend that the product variable be called oldProducts just to have a better reference of what we are saving there